notebooks/ch10/modeling_loss_functions.md

```diff
@@ -322,11 +409,11 @@ $$
 \sum_{i = 1}^{n} \theta &= \sum_{i = 1}^{n} y_i \\
```

Comment:
I think these should all be theta hats too!

Reply:
Similar to the other comment, I'll wait and see what Sam says